### PR TITLE
docs: how to join slack?

### PR DESCRIPTION
**Pull request by juzhiyuan**
I'd like to add a post under the General Documentation, users could join Slack according to 2 ways.

### CODE REVIEW

File: website/docs/general/slack.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+id: slack
+title: Slack
```
Comment (liuxiran) [inline website/docs/general/slack.md]: how about use 'how to join slack' here?

Comment (juzhiyuan) [reply website/docs/general/slack.md]: A little long 🤔

Comment (bzp2010) [reply website/docs/general/slack.md]: Can we change it to `community` or something like it? We can provide more communication way on this page.

Comment (liuxiran) [reply website/docs/general/slack.md]: > Can we change it to community or something like it? We can provide more communication way on this page.

agree

Comment (juzhiyuan) [reply website/docs/general/slack.md]: ok

**Review by bzp2010 — APPROVED** [review]
LGTM

**Review by liuxiran — APPROVED** [review]

**Pull request merged**